datajunction-server/tests/api/namespaces_test.py

```diff
@@ -10,7 +10,7 @@ def test_list_all_namespaces(client_with_examples: TestClient) -> None:
     """
     response = client_with_examples.get("/namespaces/")
     assert response.ok
-    assert response.json() == [
+    assert set(response.json()) == {
```

Comment:
Was the `GET /namespaces/` endpoint returning duplicates? If so that should be fixed server side.

Reply:
I suspect this is because the endpoint doesn't return them in any particular order. It shouldn't be returning duplicates since it's just a simple select all from the namespaces table and there's a unique constraint.

Reply:
Yeah, no duplicates, just a different order.